src/txn/txn_rollback_to_stable.c

```diff
@@ -137,6 +71,198 @@ __txn_abort_newer_insert(
             __txn_abort_newer_update(session, ins->upd, rollback_timestamp);
 }
 
+/*
+ * __txn_abort_row_replace_upd_list --
+ *     Abort updates in the update list and replace it with the provided update
+ *
+ */
+static inline int
+__txn_abort_row_replace_upd_list(
+  WT_SESSION_IMPL *session, WT_PAGE *page, WT_ROW *rip, WT_UPDATE *upd)
```

Comment:
Both the comment and function name discuss 'abort' but it isn't clear in the function itself where anything like an 'abort' happens. We allocate and add in the update, but I don't see anything aborting. If I'm missing, please add a comment to connect it with that action, or consider renaming the function.

Reply:
Changed the function name to represent what it does.